brigade/templates/styleguide.html

```diff
@@ -151,8 +160,33 @@ <h2 class="card__title">Card title</h2>
       </div>
     </div>
 
+    <h3>Variant: card--multimedia</h3>
```

Comment:
Thanks for doing this!! :clap:
Keeping our styleguide up-to-date! :raised_hands: